styles.css

```
@@ -0,0 +1,129 @@
+* {
+    box-sizing: border-box;
+  }
+  
+  :root {
+    --color-primary: #007bff;
+    --color-base: #fff;
+    --color-light-grey: #ccd5ae;
+    --color-dark-grey: rgba(0,0,0,0.5);
+    --color-dark: #000;
+  }
+  
+  html, body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    height: 100%;
+    font-family: 'Roboto', sans-serif;
+  }
+  
+  body {
+    background-color: #f1f2f3;
+  }
+  
+  main {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: center;
+    align-items: center;
+    align-content: stretch;
```

Add some space at the top of the page
```suggestion
    align-content: stretch;
    padding-top: 50px;
```